Packs/FeedUnit42v2/Integrations/FeedUnit42v2/FeedUnit42v2.py

```diff
@@ -698,7 +719,10 @@ def main():
 
     except Exception as err:
         demisto.error(traceback.format_exc())  # print the traceback
-        return_error(err)
+        if 'substring not found' in str(err):
+            return_error("Failed parsing fetch-indicators response")
```

Comment:
When do you expect this to happen?

Reply:
When the str.index function raises the ValueError it raises a string 'substring not found' but since i wrap this error, then maybe after thinking about it, it would be better to check if the error message i wrapper occurred instead.